all.json

```diff
@@ -14,6 +14,17 @@
 		"1inch-network.net",
 		"1inch-wallet.org",
 		"1inch.tk",
+		"accessbot.net",
+		"anchorprotocolapp.info",
+		"apinetworkbridge.net",
+		"col1ab.land",
+		"kcoins.exchange",
+		"poilegon-technology.org",
```

Comment:
Not sure I get this one? Food-related site?

Reply:
It is a clear imitation of polygon.

fake misspelled
`poilegon-technology.org`
real
`polygon.technology`

You either need to have a specific finger print (i.e. mobile device) and access it from a given geographical location in order to get the phishing content (i think it is in /assets/)

You need to fuzz it to get to the phish and it is not worth the effort. Block on sight is the best. 

Scam sites would often display random junk content, before they get weaponized, or a scam landing page.
If you don't know where the phishing is you may miss it.

This is done to confuse automated scanners, parsers and threat annalists.

![image](https://user-images.githubusercontent.com/49607867/168003034-65208080-23fa-40f5-9ad1-3b21e1d87d61.png)
![image](https://user-images.githubusercontent.com/49607867/168003068-02a2635d-9c9a-4857-880d-665f6ac1b669.png)